### ISSUE TITLE
Support for manual synchronization 

### ISSUE THREAD

**Issue by probar** [calzoneman/sync#388]
Iframe embeds are a valuable source of content , but they don't support synchronization - which make the usage experience pretty bad . 

Maybe it's possible to add a little clock that will show what time the current movie is on(which could be settable by admin at beginning of the movie) ? That way people could sync the video manually - but still watch it as if it was synced ?

**Comment by calzoneman** [comment]
`iframe` embeds are sort of a last resort if nothing else will possibly work, hence I don't put too much effort into making it a viable way of sharing content (I want to encourage people to use better supported, autosynch providers whenever possible).

I will consider making a timer feature core (some channels have used Javascript to add a timer), but it will be low priority.

tl;dr the only reason iframe embeds exist is because certain channels bugged me a lot before many other providers were implemented, and it is intentional that using them is a less satisfying experience because I don't want people to use them unless there is literally no other option.

**Comment by probar** [comment]
Thanks. I will look into getting or writing a javascript.

**Comment by calzoneman** [comment]
Closing for now.
